firebase-inappmessaging/firebase-inappmessaging.gradle

```diff
@@ -139,7 +139,9 @@ dependencies {
     testImplementation "com.google.truth:truth:$googleTruthVersion"
     testImplementation 'junit:junit:4.12'
     testImplementation 'androidx.test:runner:1.2.0'
-    testImplementation 'org.robolectric:robolectric:4.2'
+    testImplementation ('org.robolectric:robolectric:4.2') {
+        exclude group: 'com.google.protobuf', module: 'protobuf-java'
```

Comment:
Why is this change required?

Reply:
This allows ProtoMarshallerClientTest.java to run - without this, we get makeImmutable() errors - I don't 100% understand why, but I think it has to do with conflicting versions of proto